olp-cpp-sdk-core/include/olp/core/client/OlpClientSettingsFactory.h

```diff
@@ -43,6 +43,13 @@ class CORE_API OlpClientSettingsFactory final {
    */
   static std::unique_ptr<thread::TaskScheduler> CreateDefaultTaskScheduler(
       size_t thread_count = 1u);
+
+  /**
+   * @brief Creates a Network instance used for all non-local requests,
+   * which is defaulted to olp::client::CurlNetwork
+   * @return An instance of Network.
+   */
+  static std::unique_ptr<thread::TaskScheduler> CreateDefaultNetwork();
```

Comment:
depends on the name of the member proposed above.
Typo `thread::TaskScheduler`-> `network2::Network`

The declaration will look like this:
```cpp
std::unique_ptr<Network> CreateDefaultRequestHandler() {
#ifdef NETWORK_HAS_CURL
  return std::make_unique<NetworkCurl>();
#endif
#ifdef NETWORK_HAS_ANDROID
  return std::make_unique<NetworkAndroid>();
#endif
#ifdef NETWORK_HAS_IOS
  return std::make_unique<NetworkIos>();
#endif
#ifdef NETWORK_HAS_WINHTTP
  return std::make_unique<NetworkWinHttp>();
#endif
  return nullptr;
}
```

Reply:
The classes are not ready yet, typo fixed.